### DISCUSSION TITLE
Does this extension work in github.dev? (vscode for web)

### DISCUSSION THREAD

**Discussion by navanjr** [znck/grammarly#417] [Q&A]
I have found this extension to work with github codespaces, but...

I want to use Grammerly with [github.dev web-based editor](https://docs.github.com/en/codespaces/the-githubdev-web-based-editor)

I have tested both v0.24.0 and v0.25.0...  no joy.

**Comment by lane-willis** [comment]
I am currently struggling with getting it to work on the github.dev web-based editor also. It has not worked for me in a couple of weeks. I keep getting this message: [Error - 2:10:31 PM] [Error in Grammarly SDK]: document is not defined.
